server/annotations.go

```diff
@@ -189,7 +233,7 @@ type newAnnotationRequest struct {
 	StartTime time.Time
 	EndTime   time.Time
 	Text      string `json:"text,omitempty"` // Text is the associated user-facing text describing the annotation
-	Type      string `json:"type,omitempty"` // Type describes the kind of annotation
```

Comment:
It was originally included in anticipation of an ambiguous “annotation scoping” feature like the one I'm implementing here. I've remove it since it doesn't quite fit the role and is unused.

I could also:

- Remove it in a separate PR
- Just leave it in

What do you think?

Reply:
If this is replacing it, then I think remove it. Makes sense to do now.